dotnet-interactive/KernelExtensions.cs

```diff
@@ -76,22 +76,13 @@ public static T UseHttpApi<T>(this T kernel, StartupOptions startupOptions)
                 {
                     if (context.Command is SubmitCode submitCode)
                     {
-                        var scriptContent =
-                            HttpApiBootstrapper.GetJSCode(new Uri($"http://localhost:{startupOptions.HttpPort}"));
-
-                        string value =
-                            script[type: "text/javascript"](
-
-                                    scriptContent.ToHtmlContent())
-                                .ToString();
-
+                        var html = HttpApiBootstrapper.GetHtmlInjection(new Uri($"http://localhost:{startupOptions.HttpPort}"));
                         context.Publish(new DisplayedValueProduced(
-                            scriptContent,
+                            null,
```

Comment:
This whole call could be replaced by `context.Display()`.

We shouldn't pass null as the value in any case.